src/redux/sagas/wallet/RetryProvider.ts

```diff
@@ -78,7 +79,8 @@ const classFactory = (
       // These methods are methods that we call inside try-catches and expect
       // to fail sometimes, so don't retry.
       if (
-        method === RetryProviderMethod.Call &&
+        (method === RetryProviderMethod.Call ||
+          method === RetryProviderMethod.EstimateGas) &&
```

Comment:
@rdig would be great to get your input on this specifically. We noticed the RetryProvider gets bypassed for specified function calls only when it calls them, but not during gas estimation.

Is it okay to also bypass on gas estimation? I can't think of a reason why not but want to double check.

Reply:
Yeah, can be bypassed, as long as it it still gets an estimate from the "regular" provider, we're all good.